bucket/quick-lint-js.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "2.3.1",
+    "description": "Find bugs in JavaScript programs.",
+    "homepage": "https://quick-lint-js.com/",
+    "license": "GPL-3.0-or-later,...",
```

Comment:
```suggestion
    "license": "GPL-3.0-or-later",
```

Reply:
I think it's incorrect to say that the quick-lint-js .exe is GPL-3.0-or-later. `quick-lint-js.exe` statically links Microsoft's CRT and some open source third-party libraries (such as Boost and simdjson). Details can be found in `share/doc/quick-lint-js/copyright` in the .zip-s.

Should I instead list all the licenses separately? I think the complete list (for the 2.3.1 release) is: GPLv3, MIT, BSL, Apache2.0, and Microsoft's CRT license.

Reply:
> Should I instead list all the licenses separately?

Yes, separated by a `|` or `,`

Reply:
Hey @strager! Any updates on this?

Reply:
I'm working on a release of quick-lint-js with slightly simpler licensing. I'll update this PR when the new release is ready.